actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -499,46 +499,44 @@ def apply_form_for_options!(object, options) # :nodoc:
       #   <% end %>
       #   # =>
       #   <form method="post">
-      #     <input type="text" name="title">
+      #     <input type="text" name="title" />
       #   </form>
       #
       #   # Adding a scope prefixes the input field names:
-      #   <%= form_with scope: :post, url: posts_path do |form| %>
+      #   <%= form_with scope: :article, url: articles_path do |form| %>
       #     <%= form.text_field :title %>
       #   <% end %>
       #   # =>
-      #   <form action="/posts" method="post">
-      #     <input type="text" name="post[title]">
+      #   <form action="/articles" method="post">
+      #     <input type="text" name="article[title]" />
       #   </form>
       #
       #   # Using a model infers both the URL and scope:
-      #   <%= form_with model: Post.new do |form| %>
+      #   <%= form_with model: Article.new do |form| %>
       #     <%= form.text_field :title %>
       #   <% end %>
       #   # =>
-      #   <form action="/posts" method="post">
-      #     <input type="text" name="post[title]">
+      #   <form action="/articles" method="post">
+      #     <input type="text" name="article[title]" />
       #   </form>
-      #
```

Comment:
Any particular reason to removing the blank lines? We should probably keep them for better readability between examples (there's a couple more)

Reply:
My mistake... I'll fix.